web/src/settings_invites.ts

```diff
@@ -185,6 +187,7 @@ function do_revoke_invite({
             $revoke_button.text($t({defaultMessage: "Error!"}));
         },
         success() {
+            modals.close_active_if_any();
```

Comment:
It is fine to use `dialog_widget.close` here. I just meant to not replace `confirm_dialog.launch` with `dialog_widget.launch`.